src/syscall_mock/mod.rs

```diff
@@ -158,3 +167,47 @@ impl SyscallMock {
         }
     }
 }
+
+#[cfg(test)]
+mod syscall_mock {
+    use super::*;
+    use crate::protocol::Step;
+    use crate::{Context, R};
+    use nix::unistd::Pid;
+    use test_utils::trim_margin;
+
+    #[test]
+    fn resolves_full_path_of_unmocked_commands() -> R<()> {
+        let unmocked_commands = vec![Vec::from("cp")];
+        let syscall_mock = SyscallMock::new(
+            &Context::new()?,
+            Pid::from_raw(123),
+            Protocol::empty(),
+            unmocked_commands.as_slice(),
+        );
+        assert!(syscall_mock.unmocked_commands[0].is_absolute());
+        Ok(())
+    }
```

Comment:
I don't think we should test that `SyscallMock::new` performs this translation. That's an implementation detail that we ideally would want to be able to refactor without having the tests fail. Or was there a special corner-case or other reason to add this test?

Reply:
Good point; this is now tested with an integration test, `looks_up_unmocked_command_executable_in_path`.